### ISSUE TITLE
Update command to install package in the README.md

### ISSUE THREAD

**Issue by klangner** [lovasoa/elm-csv#2]
When I try to install this library with the command:
```
elm package install gummesson/elm-csv
```

I got error:
```
Error: I cannot find a set of packages that works with your constraints.

--> There are no versions of gummesson/elm-csv that work with Elm 0.18.0. Maybe
    the maintainer has not updated it yet.
```

I guess the problem is with the README.
Since when I install with the command:
```
elm package install lovasoa/elm-csv
```
It works.

**Comment by lovasoa** [comment]
You are trying to install a different package. This package is named lovasoa/elm-csv and not gummesson/elm-csv. 

23 марта 2017 г. 15:02:27 GMT+03:00, Krzysztof Langner <notifications@github.com> пишет:
>When I try to install this library with the command:
>```
>elm package install gummesson/elm-csv
>```
>
>I got error:
>```
>Error: I cannot find a set of packages that works with your
>constraints.
>
>--> There are no versions of gummesson/elm-csv that work with Elm
>0.18.0. Maybe
>    the maintainer has not updated it yet.
>```
>
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/lovasoa/elm-csv/issues/2

-- 
Простите за краткость, создано в K-9 Mail.

**Comment by klangner** [comment]
Thanks for the fast response.
Yes I notice it.

I think it is worth to change this in the README.md
https://github.com/lovasoa/elm-csv/blob/master/README.md

Since I used this command based on this file.

**Comment by lovasoa** [comment]
Fixed by ae1046fbed655ccaf95896889ab97c1478bc7170
